docs/getting-started.md

```diff
@@ -190,6 +190,7 @@ Now that you're setup, if you can run it on the command line, you can run it in
 ### Running AI vs AI
 1. Open your terminal, `cd` into `Software/src`
 2. Run `./software/run_ai_vs_ai.sh interface_name`, using the same interface as from [above](#running-our-ai-simulator-or-robot-diagnostics)
+3. If AI vs AI is not running, the networking ports are likely the cause. Run `./../../environment_setup/configure_ai_vs_ai.sh` to open the ports.
```

Comment:
nit: I think giving the aboluste path for `configure_ai_vs_ai.sh` will be more clear.

Reply:
Addressed.